batch/batch/aioclient.py

```diff
@@ -105,11 +106,21 @@ def __init__(self, client, id, attributes):
 
 
 class BatchClient:
-    def __init__(self, session, url=None):
+    def __init__(self, session, url=None, token_file=None, token=None, headers=None):
         if not url:
             url = 'http://batch.default'
         self.url = url
         self._session = session
+        if token is None:
+            token_file = (token_file or
```

Comment:
I think the existing jet mounted path (see notebook2 pod spec) should be one of the standard paths

Reply:
Nvm this isn’t the  userjwt